test/tests/smoke_test.rs

```diff
@@ -50,4 +99,53 @@ fn smoke_test() {
 | |__| (_| | | | |_) | |_| | | (_| | |  _ < | |
  \____\__,_|_|_| .__/ \__|_|  \__,_| |_| \_\|_|"#,
     );
+
+    const TEST_ONLY_GET_LDEV_CERT: u32 = 0x4345524c; // "CERL"
+    const TEST_ONLY_GET_FMC_ALIAS_CERT: u32 = 0x43455246; // "CERF"
+
+    let ldev_cert_der = hw
+        .mailbox_execute(TEST_ONLY_GET_LDEV_CERT, &[])
+        .unwrap()
+        .unwrap();
+    let ldev_cert = openssl::x509::X509::from_der(&ldev_cert_der).unwrap();
+    let ldev_cert_txt = String::from_utf8(ldev_cert.to_text().unwrap()).unwrap();
+
+    // To update the ldev cert testdata:
+    // std::fs::write("tests/smoke_testdata/ldevid_cert.txt", &ldev_cert_txt).unwrap();
+    // std::fs::write("tests/smoke_testdata/ldevid_cert.der", &ldev_cert_der).unwrap();
+
+    assert_eq!(
+        ldev_cert_txt.as_str(),
+        include_str!("smoke_testdata/ldevid_cert.txt")
+    );
+    assert_eq!(
+        ldev_cert_der,
+        include_bytes!("smoke_testdata/ldevid_cert.der")
+    );
+
+    assert!(
+        ldev_cert.verify(&idevid_pubkey).unwrap(),
+        "ldev cert failed to validate with idev pubkey"
+    );
+
+    let ldev_pubkey = ldev_cert.public_key().unwrap();
+
+    println!("ldev-cert: {}", ldev_cert_txt);
+
+    let fmc_alias_cert = hw
+        .mailbox_execute(TEST_ONLY_GET_FMC_ALIAS_CERT, &[])
+        .unwrap()
+        .unwrap();
+    let fmc_alias_cert = openssl::x509::X509::from_der(&fmc_alias_cert).unwrap();
+
+    // TODO: (DO NOT SUBMIT) Validate the contents / measurements in the fmc_alias_cert.
```

Comment:
The rest LGTM. I'll come back when this is implemented.

I also think it would be fine to open an issue and take care of this in a follow-up